assets/js/base/components/totals/totals-item/index.js

```diff
@@ -3,19 +3,25 @@
  */
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import FormattedMonetaryAmount from '@woocommerce/base-components/formatted-monetary-amount';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
 
-const TotalsItem = ( { className, label, value, description } ) => {
+const TotalsItem = ( { className, currency, label, value, description } ) => {
 	return (
 		<div
 			className={ classnames( 'wc-block-totals-table-item', className ) }
 		>
 			<span className="wc-block-totals-table-item__label">{ label }</span>
-			<span className="wc-block-totals-table-item__value">{ value }</span>
+			<FormattedMonetaryAmount
+				className="wc-block-totals-table-item__value"
+				currency={ currency }
+				displayType="text"
+				value={ value }
+			/>
```

Comment:
Noice 👍 - I forgot about this component that Mike had created earlier in the cycle - glad to see it used here!